apis/python/src/tiledbsoma/tiledb_object.py

```diff
@@ -49,6 +53,29 @@ def __init__(
 
         self.metadata = MetadataMapping(self)
 
+    def _default_ctx(self) -> tiledb.Ctx:
+        """
+        The TileDB context used when no other is supplied. Must have good defaults for positive
+        out-of-the-box UX.
+        """
+
+        cfg = {}
+
+        # Crucial for reducing number of HTTP requests for tiledb-cloud URIs.
+        # With default (small) value, it can take many HTTP-request round trips to satisfy
+        # a single query.
+        cfg["py.init_buffer_bytes"] = 4 * 1024**3
```

Comment:
Nit:  shouldn't this be managed centrally by SOMAReader (C++ layer)?

Reply:
For `main` yes I suppose so. We don't have _all_ read paths going through C++, but yes, all _bulk_ read paths should. (Remaining reads through TileDB-Py are metadata etc.) So yes this can go away.